<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1012</width>
    <height>805</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_showimage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>241</width>
      <height>241</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>691</width>
      <height>271</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="histo">
     <item>
      <widget class="QLabel" name="label">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Input Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_showsecond">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>241</width>
      <height>241</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Output Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_add">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>640</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>ADD</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>640</y>
      <width>241</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_sub">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>640</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>SUB</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_multi">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>640</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>MULTIPLE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_avg">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>640</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>AVG</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_func">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>640</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>f(x,y)-f(x-1,y)</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>350</y>
      <width>691</width>
      <height>271</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="histo_2">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1012</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuhwq">
    <property name="title">
     <string>file</string>
    </property>
    <addaction name="actionopen_image_2"/>
   </widget>
   <addaction name="menuhwq"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionopen_image">
   <property name="text">
    <string>open image</string>
   </property>
  </action>
  <action name="actionopen_image_2">
   <property name="text">
    <string>open image</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
